core/frontend/src/render/RenderSystem.ts

```diff
@@ -14,6 +14,7 @@ import { SkyBox } from "../DisplayStyleState";
 import { imageElementFromImageSource } from "../ImageUtil";
 import { IModelApp } from "../IModelApp";
 import { IModelConnection } from "../IModelConnection";
+import { ToolAdmin } from "../imodeljs-frontend";

```

Comment:
Did your editor change this? Don't import from imodeljs-frontend inside imodeljs-frontend - import from ./tools/ToolAdmin.

Reply:
Yes... Studio Code has option to do this that I must have clicked.  Fixed.